src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
+@import "../styles/nav.scss";
+@import "../styles/card.scss";
+@import "../styles/page.scss";
+@import "../styles/stars.scss";
+@import "../utils/variables.scss";
```

Comment:
move @import "../utils/variables.scss"; to the top of this file and reset.scss as well